js/deer-render.js

```diff
@@ -204,6 +204,255 @@ DEER.TEMPLATES.shadow = (obj, options = {}) => {
+                }
+            })
+            .catch(err => { })
+        }
+    }
+}
+
+DEER.TEMPLATES.managedStatus = (obj, options = {}) => {
+    if (!obj['@id']) { return null }
+
+    return {
+        html: `looking up record info... `,
+        then: (elem) => {
+            fetch("http://store.rerum.io/v1/id/61ae693050c86821e60b5d13")
+            .then(response => response.json())
+            .then(coll => {
+                //If you want to know who placed it, you have to check the __rerum.creator on the comment Annotation
+                //Should we allow Reviewers/Curators to remove it while they are here?
+                if(coll.itemListElement.filter(record => record["@id"] === obj['@id']).length){
+                    elem.innerHTML = `✔ Submitted For Review`
+                    elem.classList.remove("bg-error")
+                    elem.classList.add("bg-success")
+                }
+                else{
+                    let drawer = `
+                        <div class="statusDrawer is-hidden"> 
+                            <div style="margin-top: 1em;"> 
+                                <p> This will alert reviewers to look at this item.  You may add a comment below. </p>
+                                <textarea class="statusCommentText" style="margin-top: 1em;"></textarea>
+                                <input class="notifyReviewerBtn success" type="button" value="Notify Reviewers" style="margin-top: 1em;" />
+                            </div>
+                        </div>
+                    `
+                    elem.innerHTML = `Submit For Review ❕ ${drawer}`
+
+                    elem.addEventListener("click", e => {
+                        if(e.target.id !== "managedStatus"){
+                            //Do not close when users click on internal items
+                            return
+                        }
+                        if(elem.querySelector(".statusDrawer").classList.contains("is-hidden")){
+                            elem.querySelector(".statusDrawer").classList.remove("is-hidden")
+                        }
+                        else{
+                            elem.querySelector(".statusDrawer").classList.add("is-hidden")
+                        }
+                    })
+
+                    elem.querySelector(".notifyReviewerBtn").addEventListener("click", e => {
+                        if(!userHasRole(["dunbar_user_curator","dunbar_user_contributor","dunbar_user_reviewer"])) { return alert(`This function is limited to contributors, reviewers, and curators.`)}
+                        let proceed = confirm("This action is connected with you username.  Click OK to proceed and add your note.")
+                        if(!proceed){return}
+                        let url = DEER.URLS.CREATE
+                        let method = "POST"
+                        const commentAnno = {
+                            "@context": "http://purl.org/dc/terms",
```

Comment:
`http://www.w3.org/ns/anno.jsonld`

Reply:
None of the other code uses that context.  I tried to keep the same context happening everywhere else.  We already know we need to bulk update a bunch of this data and I didn't just want to switch in the middle of realizing what was happening.

Examples
http://store.rerum.io/v1/id/62794f401d974d1311abd6d0
http://store.rerum.io/v1/id/61ae30fa50c86821e60b5d0e